### PR TITLE
Feature/owner login

### PR DESCRIPTION
**Pull request by Jina-n11**

### CODE REVIEW

File: owner/src/main/java/org/the_chance/honeymart/ui/base/BaseUiEffect.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package org.the_chance.honeymart.ui.base
+
+ interface BaseUiEffect
```
Comment (AzizaHelmy) [inline owner/src/main/java/org/the_chance/honeymart/ui/base/BaseUiEffect.kt:3]: why this interface now?

File: owner/src/main/java/org/the_chance/honeymart/ui/login/LoginScreen.kt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                    hint = stringResource(org.the_chance.design_system.R.string.email),
+                    iconPainter = painterResource(id = org.the_chance.design_system.R.drawable.ic_email),
+                    onValueChange = listener::onEmailInputChange,
+                    errorMessage = when (state.emailState) {
+                        ValidationState.BLANK_EMAIL -> "email cannot be blank"
+                        ValidationState.INVALID_EMAIL -> "Invalid email"
+                        else -> ""
+                    },
+                )
+                Spacer(modifier = Modifier.height(MaterialTheme.dimens.space16))
```
Comment (AzizaHelmy) [inline owner/src/main/java/org/the_chance/honeymart/ui/login/LoginScreen.kt:114]: avoid to use this spacer to add space by this approach, use padding insted

File: owner/src/main/java/org/the_chance/honeymart/ui/login/LoginScreen.kt [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+                text = stringResource(org.the_chance.design_system.R.string.mart),
+                style = Typography.displayMedium.copy(color = Color.Black)
+
+            )
+        }
+    }
+}
+
+
+//@Preview(name = "Tablet", device = Devices.TABLET, showSystemUi = true)
```
Comment (AzizaHelmy) [inline owner/src/main/java/org/the_chance/honeymart/ui/login/LoginScreen.kt:201]: if you don't need to this code, remove it

File: owner/src/main/java/org/the_chance/honeymart/ui/login/LoginUiEffect.kt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.the_chance.honeymart.ui.login
+
+import org.the_chance.honeymart.ui.base.BaseUiEffect
+
+
+sealed class LoginUiEffect : BaseUiEffect {
```
Comment (AzizaHelmy) [inline owner/src/main/java/org/the_chance/honeymart/ui/login/LoginUiEffect.kt:6]: why this class?

File: owner/src/main/res/drawable/auth_image.png
Comment (AzizaHelmy) [inline owner/src/main/res/drawable/auth_image.png:1]: we need to export it as svg insted

**Review by AzizaHelmy — APPROVED** [review]

**Review by mahmmedn19 — APPROVED** [review]

**Pull request merged**